onedal/spmd/decomposition/incremental_pca.py

```diff
@@ -0,0 +1,110 @@
+# ==============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from daal4py.sklearn._utils import get_dtype
+
+from ...datatypes import _convert_to_supported, from_table, to_table
+from ...decomposition import IncrementalPCA as base_IncrementalPCA
+from ...utils import _check_array
+from .._base import BaseEstimatorSPMD
+
+
+class IncrementalPCA(BaseEstimatorSPMD, base_IncrementalPCA):
```

Comment:
This is common comment for all online or spmd algos.
Could you please add minimal dosctrics just to recognize what is it and where to use.
Examples inside the docstrings would be prefecty